wiki/Difficulties/es.md

```diff
@@ -0,0 +1,64 @@
+# Dificultades
+
+*No debe confundirse con: [Beatmaps](/wiki/Beatmaps).*
+
+*Ver también: [Criterios de clasificación § Mapset](/wiki/Ranking_Criteria/#mapset).*
+
+La dificultad de un beatmap *describe* el nivel de habilidad necesario para completarlo.
+
+## Clasificación por estrellas
+
+![Rangos de clasificación por estrellas](SR_range.png "Rangos de clasificación por estrellas")
+
+Desde los [Criterios de clasificación](/wiki/Ranking_Criteria), una calificación por estrellas de:
+
+- ¡1.50 estrellas y menos es Easy! [Icono Easy](/wiki/shared/diff/easy-s.png "cono Easy")
```

Comment:
That exclamation point was not part of this sentence, it was part of the markdown syntax for an image
`- ¡1.50 estrellas y menos es Easy! [Icono Easy](/wiki/shared/diff/easy-s.png "cono Easy")`
:arrow_down: 
`- 1.50 estrellas y menos es Easy ![Icono Easy](/wiki/shared/diff/easy-s.png "cono Easy")`
(please repeat for the others below